mlir/test/Target/LLVMIR/Import/constant.ll

```diff
@@ -47,6 +47,16 @@ define void @undef_constant(i32 %arg0) {
 
 ; // -----
 
+; CHECK-LABEL: @poison_constant
+define void @poison_constant(double %arg0) {
```

Comment:
Nit: This is not really related to the remainder of this PR. Still, we did not test this before, so it might make sense to keep it here?

Reply:
Yes I added it intentionally since it was missing.